templates/partials/_juju-supports.html

```diff
@@ -0,0 +1,158 @@
+<section class="p-strip">
+  <div class="row">
```

Comment:
any reason you use `row>col-12` instead of `u-fixed-width`?

Reply:
Personal preference - when I'm using nested rows/columns I like that it's clear from the markup how many columns the parent has.